src/js/component.js

```diff
@@ -1524,6 +1498,51 @@ class Component {
     return id;
   }
 
+  /**
+   * Request an animation frame, but only one named animation
+   * frame will be queued. Another will never be added until
+   * the previous one finishes.
+   *
+   * @param {string} name
+   *        The name to give this requestAnimationFrame
+   *
+   * @param  {Component~GenericCallback} fn
+   *         A function that will be bound to this component and executed just
+   *         before the browser's next repaint.
+   */
+  requestNamedAnimationFrame(name, fn) {
+    if (this.namedRafs_.has(name)) {
+      return;
+    }
+
+    this.clearTimersOnDispose_();
+    fn = Fn.bind(this, fn);
+
+    const id = this.requestAnimationFrame(() => {
+      fn();
+      if (this.namedRafs_.has(name)) {
+        this.namedRafs_.delete(name);
+      }
+    }, false);
```

Comment:
Why is `false` being passing to RAF? I'm not aware of a second parameter for RAF.

Reply:
good catch, I think I originally had a second parameter being passed to `this.requestAnimationFrame` but refactored it out. I guess I left it's usage here though.